packages/react-instantsearch-hooks-dom/src/ui/InfiniteHits.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { Hit } from 'instantsearch.js';
+import type { JSXElementConstructor } from 'react';
```

Comment:
Could we reference this from `React` (`React.JSXElementConstructor`) to keep the imports terse?

Totally optional.